portal-api/src/v1/validation/validateIds.js

```diff
@@ -60,6 +60,22 @@ const isValidMonth = (month) => Number.isInteger(month) && month >= 1 && month <
  */
 const isValidYear = (year) => Number.isInteger(year) && year >= 2000 && year <= 2100;
 
+/**
+ * isValidReportPeriod
+ * checks if the supplied report period is a valid report period object
+ * @param {import('../../types/utilisation-reports').ReportPeriod} reportPeriod
```

Comment:
Should type this as `unknown` as a prompt to check that the values exist as is expected of the `ReportPeriod` type.